shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/uimodel/QuestionUiModel.kt

```diff
@@ -0,0 +1,26 @@
+package co.nimblehq.avishek.phong.kmmic.presentation.uimodel
+
+import co.nimblehq.avishek.phong.kmmic.domain.model.AnswerInput
+import co.nimblehq.avishek.phong.kmmic.domain.model.Question
+import co.nimblehq.avishek.phong.kmmic.domain.model.QuestionDisplayType
+
+data class QuestionUiModel(
+    val id: String,
+    val step: String,
+    val displayOrder: Int,
+    val displayType: QuestionDisplayType,
+    val questionTitle: String,
+    val answers: List<SurveyAnswerUiModel>,
+    var userInputs: Set<AnswerInput>
```

Comment:
Just out of curiosity, why don't we use `List<AnswerInput>` here?

Reply:
Using Set will be easier to update the view, especially in the case NPS and radio button answers